corehq/apps/export/models/new.py

```diff
@@ -920,6 +933,53 @@ def get_filters(self):
         return []
 
 
+class SMSExportInstance(ExportInstance):
+    type = SMS_EXPORT
+    identifier = None
+    name = "Messages"
+
+    @classmethod
+    def _new_from_schema(cls, schema):
+        columns = [
+            ("Contact Type", "couch_recipient_doc_type", DOC_TYPE_TRANSFORM),
+            ("Contact ID", "couch_recipient", None),
+            ("Timestamp", "date", None),
+            ("Owner Name", "couch_recipient", OWNER_ID_TRANSFORM),
+            ("Phone Number", "phone_number", None),
+            ("Direction", "direction", None),
+            ("Message", "text", None),
+            ("Type", "workflow", WORKFLOW_TRANSFORM),
+        ]
+        main_table = TableConfiguration(
+            label='Messages',
+            path=MAIN_TABLE,
+            selected=True,
+            columns=[
+                ExportColumn(
+                    label=col[0],
+                    item=ExportItem(
+                        path=[PathNode(name=col[1])],
+                        transform=col[2]
+                    ),
+                    selected=True
+                )
+                for col in columns
+            ],
+        )
+        if schema.include_metadata:
+            main_table.columns.append(ExportColumn(
+                label="Message Log ID",
+                item=ExportItem(
+                    path=[PathNode(name='_id')]
```

Comment:
yeah SMS models are only in postgres now.  i'd prefer to keep couch_id as the uuid for an SMS though because we may need to shard it some day and then we couldn't use the integer id anymore

i don't know if _id in the ES doc will be the right thing, it might be if it gets populated by [this](https://github.com/dimagi/commcare-hq/blob/2e0d3a06d394753b188b3fbcdc97b3aaffbaf901/corehq/pillows/sms.py#L47)